docs/products/mysql/concepts/high-availability-mysql.md

```diff
@@ -0,0 +1,112 @@
+---
+title: High availability of Aiven for MySQL®
+sidebar_label: High availability
+---
+
+Aiven for MySQL® is available on a variety of plans, offering different levels of high availability. The selected plan defines the features available, and a summary is provided in the table below:
+
+| Plan         | High availability features                                      | Backup history | Uptime  |
+| ------------ | --------------------------------------------------------------- | -------------- | ------- |
+| **Hobbyist** | Single node with limited availability                           | 2 days         | 98.5 %  |
+| **Startup**  | Single node with limited availability                           | 2 days         | 99 %    |
+| **Business** | Two nodes (primary + standby: basic high availability)          | 14 days        | 99.9 %  |
+| **Premium**  | Three nodes (primary + 2 x standby: stronger high availability) | 30-day         | 99.99 % |
+
+## About primary and standby nodes
+
+Aiven's Business and Premium plans offer primary nodes and standby nodes. A standby
+service is useful for multiple reasons:
+
+-   Provides another physical copy of the data in case of hardware,
+    software, or network failures
+-   Typically reduces the data loss window in disaster scenarios
+-   Provides a quicker database time to restore with a controlled
+    failover in case of failures, as the standby is already installed,
+    running, and synchronised with the data
+-   Can be used for read-only queries to reduce the load on the primary
+    server
+
+## Failure handling
+
+### Minor failures
```

Comment:
This section describes resuming service after a minor failure by restarting a crashed process.  In the presence of a standby node, I would not do this bvecause crash recovery of MySQL can take time, and then MySQL starts with a cold cache.  IMHO and when there is a standby node, I would prefer failing-over.

Reply:
@alexole , could you please comment?

Reply:
For MySQL we do the fail-over in case of a crash loop, single crash does not lead to failover.